lib/storage/src/content_manager/consensus_manager.rs

```diff
@@ -118,6 +118,12 @@ impl<C: CollectionContainer> ConsensusManager<C> {
         }
     }
 
+    pub fn report_snapshot(&self, peer_id: u64, status: impl Into<SnapshotStatus>) {
+        let _ = self
+            .propose_sender
+            .send(ConsensusOperations::report_snapshot(peer_id, status));
```

Comment:
Is this completely swallowing a potential error?

Reply:
Well, yeah, but:

- it's a sender error
  - so, it only happens when the receiver side is closed
  - which means consensus thread is toast
  - which means we're either shutting down, or we have bigger problems than this one error and it won't help us much
- currently `report_snapshot` is only called __by the subtask the consensus thread spawns itself__
- and given all that, I was lazy to properly propagate this error :D

I'll refactor the method itself to return a `Result`, but IMO, it's not even worth it to log this error, as it will only add noise.